sklearn/metrics/tests/test_classification.py

```diff
@@ -160,7 +144,6 @@ def test_precision_recall_f1_score_binary():
                             (1 + 2 ** 2) * ps * rs / (2 ** 2 * ps + rs), 2)
 
 
-@ignore_warnings
```

Comment:
If you've actually checked it doesn't raise warnings (by setting the warning response to 'error') then it's fine to remove, though it'd still be simpler if it were isolated in a quick PR.